libraries/testbot/bots/dialogBot.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityHandler } = require('botbuilder');
+
+class DialogBot extends ActivityHandler {
```

Comment:
Add unit tests for DialogBot for parity with dotnet

Reply:
we'll have to leave this one as debt for this release, most of the important code is covered by the test for dialogAndWelcomeBot.